ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -34,6 +34,11 @@ let initialize_info : InitializeResult.t =
     CompletionOptions.create ~triggerCharacters:[ "."; "#" ]
       ~resolveProvider:true ()
   in
+  let signatureHelpProvider =
+    SignatureHelpOptions.create
+      ~triggerCharacters:[ " "; "~"; "?"; ":"; "("; "["; "{" ]
```

Comment:
`{` is for records in the future right? Why `[` though?

Reply:
I thought I noticed an interaction with the function signature help, but I checked again and it doesn't make a difference. Removed.